tap_egencia/streams.py

```diff
@@ -16,11 +16,14 @@
 
 
 from tap_egencia.schemas.get_transactions import (
-    metadataObject,
```

Comment:
We should expose these changes as a separate stream, unless we have confirmation from DE that they're okay with getting the individual transactions instead of the raw responses (granted the extra data in the full response is not useful to anyone internally in any way I don't think, but let's make sure they're on the same page since right now they're expecting the full response)

Reply:
Reach out to John Timeus on teams and see what he says. If he's good with getting the transaction-specific stream then we're good to go. If he wants the really raw responses, then we should have a separate stream.

Reply:
Oh just saw this, I separated it into another stream just now.